src/components/Particles/smoke/SmokeParticles.jsx

```diff
@@ -0,0 +1,27 @@
+import { SmokeParticle } from "./SmokeParticle";
+
+export const SmokeParticles = ({ driftRight, driftLeft }) => {
+
+
+  return (
+    <group>
+      {/* bottom left */}
+      <group>
+        <SmokeParticle
+          position={[-0.6, 0.05, 0.5]}
+          png="./particles/fire_02.png"
+          leftDrift={driftLeft}
+          rightDrift={driftRight}
+          delay={200}
+        />
+        <SmokeParticle
+          position={[0.6, 0.05, 0.5]}
+          png="./particles/fire_01.png"
+          leftDrift={driftLeft}
+          rightDrift={driftRight}
+          delay={200}
+        />
+      </group>
+    </group>
```

Comment:
group in group? what is the first group used for?

Reply:
My fault